src/Drupal/Commands/core/MigrateRunnerCommands.php

```diff
@@ -493,8 +494,14 @@ public function resetStatus(string $migrationId): void
      * @param string $migrationId
      *   The ID of the migration.
      *
+     * @option idlist Comma-separated list of IDs to import. As an ID may have more than one column, concatenate the columns with the colon ':' separator.
+     *
      * @usage migrate:messages article
      *   Show all messages for the <info>article</info> migration
+     * @usage migrate:messages article --idlist=5
+     *   Import the article record with source ID 5.
+     * @usage migrate:messages node_revision --idlist=1:2,2:3,3:5
```

Comment:
Can we convert one of the IDs as strings and show how to properly quote an ID containing `:`. Also we can explain this is usage description.